### DISCUSSION TITLE
Domyos Bike 900 + Ubuntu 21.10 + Zwift

### DISCUSSION THREAD

**Discussion by burton56** [cagnulein/qdomyos-zwift#693] [Q&A]
Hello,
i'm trying to use qdomyos-zwift package in configuration like in topic title. I'm starting app via terminal sudo ./qdomyos-zwift
When i start pedaling on a bike i can see in the app that it has connected to the bike. Then i'm trying to search for sensors in Zwift on my phone but it can't find any kind of sensor. In bluetooth mamanger on phone i can see DomyosBridge but when i try to pair with it, it says that app is needed. What am i  missing?

**Comment by cagnulein** [comment]
Hi, first of all you don't have to pair at all with your phone.
Which dongle are you usin on ubuntu? Not all the dongles are supporting the bridge functionality
There is a dongle advised in the wiki
Let me know

**Reply by burton56** [reply]
First of all, thanks for repy.
I'm using bluetooth built in Dell Latitude 5590 laptop. It is Intel Crop. Wireless 8265/8275 adapter.

**Reply by cagnulein** [reply]
@burton56 you should give it a try to https://www.amazon.com/dp/B07V1SZCY6?psc=1&ref=ppx_yo2_dt_b_product_details&fbclid=IwAR1FvtVf-9PJzFw1qcjE1xJ-uQ-z-oe-VQAJYFm1Y-qq8v_P_LijkYFAGOY

**Reply by burton56** [reply]
I have some other bt dongles. I'll give them a try and respond here :)

**Comment by burton56** [comment]
Ok, so i have tested and can confirm that those chips doesn't work:
Chip-id: 8087:0a2b (Intel 8265)
Chip-id: 0a12:0001 (CSR8510 descripted as UGREEN 4.0)
Chip-id: 0a12:0001 (Some old bt 2.1 dongle descripted as BM07-C08-V1)
Samsung SM-A750F.

I have successfully connected with phones: 
Samsung SM-A715
Samsung SM-F711B

**Reply by cagnulein** [reply]
did you buy also the one that i suggested? THanks for the feedbacks!

**Reply by burton56** [reply]
No, currently i'm using SM-A715 as proxy device.
